query-engine/core/src/interactive_transactions/transaction.rs

```diff
@@ -174,7 +174,7 @@ impl InteractiveTransaction {
                 operation,
                 traceparent,
             )
-            .instrument(info_span!("prisma:engine:itx_execute_single", user_facing = true))
+            .instrument(info_span!("prisma:engine:itx_execute_single"))
```

Comment:
Why are we removing `user_facing = true` here?

Reply:
because it's just noise, we already have user facing `prisma:engine:query` which is a child of `prisma:client:transaction`, this should really be an internal span